tests/event-rider.spec.js

```diff
@@ -1,73 +0,0 @@
-'use strict';
```

Comment:
seems like we got rid of a lot of tests, but coverage dropped

Reply:
Yes, I'll add more